docs/articles/templates/selectors/selector-options.md

```diff
@@ -113,8 +91,7 @@ Consider the following page:
 </html>
 ```
 
-When a selector with `visibilitycheck` enabled is tested for an element existance
-or the number of matching elements, invisible elements also count.
+When a selector with `visibilitycheck` enabled is tested for an element existence or the number of matching elements, invisible elements also count.
```

Comment:
"an element existence"?

Reply:
invisible elements also count -> invisible elements are also counted

Reply:
*When you use a selector with `visibilitycheck` enabled to determine if an element exists or to count matching elements, TestCafe also takes into account invisible elements.*

Reply:
Ok